devops/ansible/playbook.yml

```diff
@@ -233,6 +233,9 @@
       DISPLAY: :7
     when: arbor
 
+  - name: arbor | install logo
+    get_url: url=https://arbor.kitware.com/arbor.jpg dest=/vagrant/app/arbor.jpg
```

Comment:
How about a tangelo logo if it is a build for tangeloHub? Or are we already doing that?

Sent from my iPhone

> On Apr 27, 2015, at 2:33 PM, Jeffrey Baumes <notifications@github.com> wrote:
> 
> In devops/ansible/playbook.yml:
> 
> > @@ -233,6 +233,9 @@
> >        DISPLAY: :7
> >      when: arbor
> >  
> > +  - name: arbor | install logo
> > +    get_url: url=https://arbor.kitware.com/arbor.jpg dest=/vagrant/app/arbor.jpg
> Please add a "when: arbor" here as in other steps above.
> 
> —
> Reply to this email directly or view it on GitHub.
>